tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_beacon.c

```diff
@@ -381,6 +381,7 @@ static void test_tx_invalid(void)
 	int err;
 
 	bt_mesh_test_cfg_set(&tx_cfg, 130);
+	bt_mesh_crypto_init();
```

Comment:
why is it required? bsim tests do not support mbedtls library yet.

Reply:
It looks working from what I see. Bsim tests compile with mbedtls and work. This is needed since PSA needs to be initialized before used. mesh is not enabled in this test function, but some mesh crypto API is used, therefore this function called explicitly here.